src/renderer/qssWidget/js/qssVolumeWidget.js

```diff
@@ -444,19 +446,24 @@
      * subtracted from or added to the setting's value.
      * @return {Boolean} Whether there was a change in the setting's value.
      */
-    gpii.qssWidget.volumeStepper.makeRestrictedStep = function (that, value, schema, stepMultiplier) {
-        var step = schema.divisibleBy * stepMultiplier;
+    gpii.qssWidget.volumeStepper.makeRestrictedStep = function (that, value, schema, stepMultiplier, previousValue) {
```

Comment:
Update JSDocs for this change in signature - I am assuming that this pull failed linting